src/index.html

```diff
@@ -5,10 +5,91 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
-    <link rel="stylesheet" href="./css/index.scss" />
+    <link rel="stylesheet" href="/css.81192ade.css" />
+    <script src="/css.81192ade.js"></script>
   </head>
+
   <body>
-    <div id="main"></div>
-    <script src="./js/index.js"></script>
+    <script src="/js.00a46daa.js"></script>
+    <nav>
+      <a href="/index.html">HOME</a>
+      <button class="btn">DOWNLOAD MY FULL RESUME</button>
```

Comment:
You can use `<button>` but I suggest use `<a>` as the default button style is harder to modify what you need. And if it's a button, class name shouldn't just 'btn' again.